### PR TITLE
Update DG 

### PR DESCRIPTION
**Pull request by WM71811**

### CODE REVIEW

**Review by yanbingtao — DISMISSED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -377,149 +329,60 @@ The following activity diagram summarizes what happens when a user executes a ne
...
-    
 ## \[Completed\] Reset all ingredients' levels feature
 
-Reset all ingredients' levels feature allows the user to reset all the ingredient levels to zero. It helps the user to
+tCheck allows the user to reset all the ingredient levels to zero. It helps the user to
 remove data that are no longer needed. The command is:
   
 * `i-reset-all` - Resets the ingredients' levels of all ingredients to zero.
   
 #### Completed Implementation
```
Comment (ureshiiYing) [inline docs/DeveloperGuide.md]: Change the subheader to just "Implementation"

**Review by ureshiiYing — COMMENTED** [review]
Overall LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (WM71811) [reply docs/DeveloperGuide.md]: Okay, thanks!

**Review by ureshiiYing — DISMISSED** [review]
LGTM

**Review by ureshiiYing — DISMISSED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -965,3 +847,17 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Resetting all ingredients' levels to zero
```
Comment (Persdre) [inline docs/DeveloperGuide.md:858]: This part should be before the saving data

Comment (WM71811) [reply docs/DeveloperGuide.md:858]: Okay thanks!

**Review by ureshiiYing — DISMISSED** [review]
LGTM

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM

**Pull request merged**